eng/emitter-package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "main": "dist/src/index.js",
+  "dependencies": {
+    "@typespec/http-client-csharp": "latest"
```

Comment:
Why is this needed? Should we be careful about using latest?

Reply:
You're right. I'll rip it out and just leave a note about regenerating the client in a README at the root of the `generated` folder.

Reply:
Removed.